settings.yaml

```diff
@@ -2741,7 +2741,7 @@
   units: N/A
   default value: 'Disabled'
   readonly: false
-  enumerated possible values: SBP,NMEA OUT,RTCMv3 IN,RTCMv3 OUT,EMP OUT
+  enumerated possible values: SBP,NMEA OUT,RTCMv3 IN,RTCMv3 OUT,EMP OUT,Disabled
```

Comment:
Default value takes in Disabled but it is not in the list of enumerated possible values.
Even though it auto generates, I think we should keep the Disabled as a possible value.